LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Mark Longair
```

Comment:
Is this meant to be mySociety?

Reply:
Yes, sorry - that's just what `bundle gem` defaulted to.